src/utils.js

```diff
@@ -58,12 +58,22 @@ const generateIssueContent = async (scores, renderBadge) => {
   return ejs.render(template, { scores: scoresInScope, renderBadge })
 }
 
+const makeDirectory = ({ path }) => {
```

Comment:
I believe that this should be a direct argument, as you are executing the function as `makeDirectory(scopePath)` in `action.js`

```suggestion
const makeDirectory = ( path ) => {
```